xbmc/pvr/windows/GUIViewStatePVR.cpp

```diff
@@ -77,8 +77,10 @@ void CGUIViewStateWindowPVRGuide::SaveViewState(void)
 
 CGUIViewStateWindowPVRTimers::CGUIViewStateWindowPVRTimers(const int windowId, const CFileItemList& items) : CGUIViewStatePVR(windowId, items)
 {
-  AddSortMethod(SortByLabel, 551, LABEL_MASKS("%L", "%I", "%L", ""));   // FileName, Size | Foldername, empty
-  AddSortMethod(SortByDate, 552, LABEL_MASKS("%L", "%J", "%L", "%J"));  // FileName, Date | Foldername, Date
+  SortAttribute sortAttributes(CSettings::Get().GetBool("filelists.ignorethewhensorting") ? SortAttributeIgnoreArticle : SortAttributeNone);
+  sortAttributes = (SortAttribute)((int)sortAttributes | SortAttributeIgnoreFolders);
```

Comment:
Ah, yes I see that is policy (http://kodi.wiki/view/Official:Code_guidelines_and_formatting_conventions). Sorry.
As the format is changing anyhow, I think I'll go with an int, |= and cast to SortAttribute in the AddSortMethod function call. Seems cleaner to me and also what everyone else was trying to push me towards...